samples/samples/admin/samples_test.py

```diff
@@ -141,3 +175,142 @@ def test_create_database_with_default_leader(
     out, _ = capsys.readouterr()
     assert default_leader_database_id in out
     assert default_leader in out
+
+
+@pytest.mark.dependency(name="add_and_drop_database_roles")
```

Comment:
The original test depends on another test `depends=["insert_data"]`.
Why did we need this dependency there? will it change the behavior of the test?

Reply:
That dependency is not required for this samples to work since we are adding roles which don't depend on rows to be present.
I have updated it to depend on `create_table_with_datatypes` but again GRANT does not do any validation on backend hence even if table does not exist role will be given permission.